src/components/Reports/Subscribers/Subscribers.js

```diff
@@ -33,6 +34,18 @@ const Subscribers = ({ dependencies: { dopplerApiClient } }) => {
   const [state, setState] = useState({ loading: true });
   const intl = useIntl();
   const _ = (id, values) => intl.formatMessage({ id: id }, values);
+  const tabProperties = [
+    {
+      url: `/subscribers/${email}/history`,
+      descriptionKey: 'subscriber_history.tab_description',
+      active: section === 'history',
+    },
+    {
+      url: `/subscribers/${email}/gdpr`,
+      descriptionKey: 'subscriber_gdpr.tab_description',
+      active: section === 'gdpr',
+    },
+  ];
```

Comment:
In my opinion, you should reuse sections object

Reply:
if you need, you can transform (or map) _sections_ object to generate the required structure for _Tabs_ component.

Reply:
I don't realize how to do this. _sections_ is an object and _Tabs_ component expect an array.

Reply:
But as you can see, there is relation between them, right? DRY

Let my know if you need help in the conversion.

Reply:
I will try to find a way by myself.